version.php

```diff
@@ -30,7 +30,8 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version   = 2023061900;
-$plugin->requires  = 2023042400; // Moodle 4.2, 24 April 2023, https://moodledev.io/general/releases#moodle-42
+$plugin->version   = 2023062901;
+//$plugin->requires  = 2023042400; // Moodle 4.2, 24 April 2023, https://moodledev.io/general/releases#moodle-42
```

Comment:
leave it as it is. I'll do the necessary things to keep up to date master branch and MOODLE_310_STABLE from this repo.